src/Http/Http.Results/src/AcceptedAtRoute.cs

```diff
@@ -35,10 +50,10 @@ internal AcceptedAtRoute(object? routeValues)
     /// <param name="routeValues">The route data to use for generating the URL.</param>
     internal AcceptedAtRoute(
         string? routeName,
-        object? routeValues)
+        RouteValueDictionary routeValues)
     {
         RouteName = routeName;
-        RouteValues = new RouteValueDictionary(routeValues);
+        RouteValues = routeValues ?? new RouteValueDictionary();
```

Comment:
`RouteValueDictionary routeValues` is non-nullable. I don't think it is necessary to check for null on these internal methods, since the compiler will tell us if we try passing a nullable object at the call sites.

```suggestion
        RouteValues = routeValues;
```